docs/community/tutorials-contest/2022/avalanche-theGraph-index/README.md

```diff
@@ -0,0 +1,381 @@
+## Indexing an Avalanche Local Subnet with The Graph
+
+### Introduction
+
+[Avalanche](https://www.avax.network/) is an open-source platform for launching decentralized applications and enterprise blockchain deployments in one interoperable, highly scalable ecosystem. Avalanche is the first decentralized smart contracts platform built for the scale of global finance, with near-instant transaction finality. Avalanche is a blockchain that promises to combine scaling capabilities and quick confirmation times through its Avalanche Consensus Protocol. It can process 4,500 TPS (transactions per second). For Ethereum, that number is 14 TPS.
+
+![avax](./images/1.jpeg "avax")
```

Comment:
We appreciate the sentiment. :-) But, seeing that we will publish this on our docs pages, it might be better to be cordial and avoid disparaging other projects. Can you remove this (and the other) meme?

Reply:
Images have been removed